weather_mv/loader_pipeline/ee.py

```diff
@@ -458,6 +458,17 @@ def process(self, uri: str) -> t.Iterator[AssetData]:
                 properties=attrs
             )
 
+            queue.put_nowait(asset_data)
+
+    def process(self, uri: str) -> t.Iterator[AssetData]:
+        """Opens grib files and yields AssetData."""
+        queue = Queue(maxsize=2)
```

Comment:
Why is the maxsize 2 here? Can you leave a comment with the explanation?

Reply:
I imagine the comment might explain how this is used to perform the work in a subprocess.